scripts/rebase_patches/0002-disable-APIrequestcount-controller.patch

```diff
@@ -0,0 +1,47 @@
+diff --git a/vendor/k8s.io/kubernetes/openshift-kube-apiserver/openshiftkubeapiserver/patch.go b/vendor/k8s.io/kubernetes/openshift-kube-apiserver/openshiftkubeapiserver/patch.go
+index fb66eece..046e20bc 100644
+--- a/vendor/k8s.io/kubernetes/openshift-kube-apiserver/openshiftkubeapiserver/patch.go
++++ b/vendor/k8s.io/kubernetes/openshift-kube-apiserver/openshiftkubeapiserver/patch.go
+@@ -8,7 +8,6 @@ import (
+ 	"github.com/openshift/apiserver-library-go/pkg/admission/imagepolicy/imagereferencemutators"
+ 	"github.com/openshift/apiserver-library-go/pkg/admission/quota/clusterresourcequota"
+ 	"github.com/openshift/apiserver-library-go/pkg/securitycontextconstraints/sccadmission"
+-	apiclientv1 "github.com/openshift/client-go/apiserver/clientset/versioned/typed/apiserver/v1"
+ 	configclient "github.com/openshift/client-go/config/clientset/versioned"
+ 	configv1informer "github.com/openshift/client-go/config/informers/externalversions"
+ 	quotaclient "github.com/openshift/client-go/quota/clientset/versioned"
+@@ -33,7 +32,6 @@ import (
+ 	"k8s.io/kubernetes/openshift-kube-apiserver/admission/autoscaling/managementcpusoverride"
+ 	"k8s.io/kubernetes/openshift-kube-apiserver/admission/scheduler/nodeenv"
+ 	"k8s.io/kubernetes/openshift-kube-apiserver/enablement"
+-	"k8s.io/kubernetes/openshift-kube-apiserver/filters/deprecatedapirequest"
+ 	"k8s.io/kubernetes/pkg/quota/v1/install"
+ 
+ 	// magnet to get authorizer package in hack/update-vendor.sh
+@@ -80,26 +78,6 @@ func OpenShiftKubeAPIServerConfigPatch(genericConfig *genericapiserver.Config, k
+ 	)
+ 	// END ADMISSION
+ 
+-	// HANDLER CHAIN (with oauth server and web console)
+-	deprecatedAPIClient, err := apiclientv1.NewForConfig(makeJSONRESTConfig(genericConfig.LoopbackClientConfig))
+-	if err != nil {
+-		return err
+-	}
+-	deprecatedAPIRequestController := deprecatedapirequest.NewController(deprecatedAPIClient.APIRequestCounts(), nodeFor())
+-	genericConfig.AddPostStartHook("openshift.io-deprecated-api-requests-filter", func(context genericapiserver.PostStartHookContext) error {
+-		go deprecatedAPIRequestController.Start(context.StopCh)
+-		return nil
+-	})
+-	genericConfig.BuildHandlerChainFunc, err = BuildHandlerChain(
```

Comment:
Will MicroShift ever need the patches for the OAuth metadata endpoint, impersonation backwards-compat, and console redirect that are installed here?

/cc @stlaz

Reply:
OAuth metadata endpoint would only be useful if we allowed deploying the OpenShift authentication, which I don't makes sense for our use-case, so I don't think it's going to be used.

The console redirect would likely come handy if we had console, which we don't, right?

As for the user impersonation backwards compat, IIRC it only makes sense with OpenShift authentication, so it should be safe to remove it.